solutions/azure-hub-spoke-connected-group/bicep/modules/avnm.bicep

```diff
@@ -0,0 +1,315 @@
+  dependsOn: [
+    networkManager::networkGroupProd::staticMembersSpokeOne
+    networkManager::networkGroupProd::staticMembersSpokeTwo
+  ]
+  properties: {
+    description: 'Prod spoke-to-spoke connectivity configuration (through hub)'
+    displayName: 'Prod Spoke-to-Spoke Connectivity'
+    appliesToGroups: [
+      {
+        networkGroupId: networkManager::networkGroupProd.id
+        isGlobal: 'False'
+        useHubGateway: string(deployVpnGateway)
+        groupConnectivity: 'None'
+      }
+    ]
+    connectivityTopology: 'HubAndSpoke'
+    deleteExistingPeering: 'True'
+    hubs: [ 
+      {
+        resourceId: hubVnetId
+        resourceType: 'Microsoft.Network/virtualNetworks'
+      } 
+    ]
+    isGlobal: 'False'
+  }
+}
+
+@description('This user assigned identity is used by the Deployment Script resource to interact with Azure resources.')
+resource userAssignedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2022-01-31-preview' = {
+  name: 'uai-${location}'
+  location: location
+}
+
+@description('This role assignment grants the user assignmed identity the Contributor role on the resource group.')
+resource roleAssignment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
+  name: guid(resourceGroup().id, userAssignedIdentity.name)
+  properties: {
+    roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c') // Contributor: b24988ac-6180-42a0-ab88-20f7382dd24c
+    principalId: userAssignedIdentity.properties.principalId
+    principalType: 'ServicePrincipal'
+  }
+}
+
+@description('This is the securityadmin configuration assigned to the AVNM')
+resource securityConfig 'Microsoft.Network/networkManagers/securityAdminConfigurations@2022-05-01' = {
+  name: 'sg-${location}'
+  parent: networkManager
+   properties: {
+    applyOnNetworkIntentPolicyBasedServices: ['None'] 
+    description: 'Security Group for AVNM'
+     }
+  }
+
+@description('This is the rules collection for the security admin config assigned to the AVNM')
+resource rulesCollection 'Microsoft.Network/networkManagers/securityAdminConfigurations/ruleCollections@2022-05-01' = {
+name: 'rc-${location}'
+parent: securityConfig
+properties: {
+  appliesToGroups: [
+     {
+      networkGroupId: networkManager::networkGroupAll.id
+     }
+   ]
+ }
+}
+
+@description('This is the rule is to Deny all TCP Rules')
+resource rule1 'Microsoft.Network/networkManagers/securityAdminConfigurations/ruleCollections/rules@2022-05-01' = if (deployDefaultDenySecurityAdminRules) {
+name: 'r-tcp-${location}'
+kind: 'Custom'
+parent: rulesCollection
+properties: {
+  access: 'Deny'
+  description: 'Denying TCP rules'
+  destinationPortRanges: ['20','21','22','23','69','119','161','445','512','514','873','3389','5800','5900']
+  destinations: [
+          {
+            addressPrefix: '*'
+            addressPrefixType: 'IPPrefix'
+          }
+      ]
+  direction: 'Inbound'
```

Comment:
Can you do a formatting check on this file please, looks like something got a bit off here toward the end.